### DISCUSSION TITLE
How do i get Student Developer pack help..?

### DISCUSSION THREAD

**Discussion by ayusharma03** [community/community#134996] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I am unable to recieve student developer pack.. tried multiple times

**Comment by MrDracs** [comment, marked as answer]
Go to https://education.github.com/discount_requests/application, add the email you have with your college's domain. for example: yourname@college.edu.in, then wait for github to verify it. Next steps would be to upload your student ID, and enable 2FA on your account. Hope this Helps!

**Reply by ayusharma03** [reply]
Thanks!! worked like a charm 😊

**Comment by MrDracs** [comment]
If you still dont get your application accepted, try adding full billing information for your account

**Reply by ayusharma03** [reply]
Thanks
